src/lib/import-export/export/exporters/default-exporter.ts

```diff
@@ -276,7 +276,11 @@ export class DefaultExporter extends EventEmitter implements Exporter {
 			return [];
 		}
 
-		return JSON.parse( stdout );
+		try {
+			return JSON.parse( stdout );
+		} catch ( error ) {
+			return [];
+		}
```

Comment:
I'm not totally convinced that making this error silent is the best solution. Maybe we should report it to Sentry?, or to the user? Informing the user we couldn't grab the plugins/themes information.

Reply:
It wouldn't help to pass it to Sentry, as we couldn't do much without knowing what PHP code caused the incorrect WP CLI response.

> Informing the user we couldn't grab the plugins/themes information.

Would you fail the import entirely in such a case?